public/index.html

```diff
@@ -20,6 +20,8 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="https://fonts.googleapis.com/css2?family=Fjalla+One&family=Merriweather+Sans:wght@300&display=swap"
+      rel="stylesheet">
```

Comment:
request(nb): Would you mind taking the fonts work out and doing it in another PR? I think this PR may be a lot, and I don't want to hold up other work.